components/form/Location.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState, FC } from 'react';
+import Button from '~/components/Button';
+import LocationInput from '~/components/LocationInput';
+
+import { useFormContext } from '~/lib/context/form_context';
+import { originalLat, originalLng } from '~/lib/utils';
+import { zoomLevelThreshold } from '~/lib/utils';
+interface Props {
+  nextPageCallback: () => void;
+}
+
+const Location: FC<Props> = ({ nextPageCallback }) => {
+  const { siteCoordinates, setSiteCoordinates } = useFormContext();
+  const [isSubmissionEnabled, setIsSubmissionEnabled] = useState(false);
+
+  const onClick = () => {
+    nextPageCallback();
+  };
+
+  // The map should zopm into the initial coordinates if they were entered by the user
+  const shouldZoomIntoOriginal =
+    originalLat !== siteCoordinates.latitude ||
+    originalLng !== siteCoordinates.longitude;
+
+  const title = shouldZoomIntoOriginal
+    ? 'Is this location exact?'
+    : 'Where is your solar panel located?';
+
+  return (
+    <>
+      <div
+        className="flex flex-col gap-2 relative h-[calc(100vh-var(--nav-height))] w-screen bg-mapbox-gray-1000"
+        id="rootDiv"
+      >
+        <div className="flex flex-col justify-end h-16 pl-3"></div>
+        <div
+          className="self-center md:w-7/12 w-full h-4/6"
+          id="mapboxInputWrapper"
+        >
+          <h1 className="font-medium md:text-3xl text-xl text-ocf-gray pl-3 md:pl-0">
+            {title}
+          </h1>
+          <LocationInput
+            shouldZoomIntoOriginal={shouldZoomIntoOriginal}
+            originalLat={siteCoordinates.latitude}
+            originalLng={siteCoordinates.longitude}
+            setIsSubmissionEnabled={setIsSubmissionEnabled}
+            setMapCoordinates={setSiteCoordinates}
+            zoomLevelThreshold={zoomLevelThreshold}
+            initialZoom={4}
+            canEdit={true}
+          />
+        </div>
+        <div className="md:hidden flex justify-center md:justify-end md:mt-20 mt-10 self-center w-full h-14">
+          <Button disabled={!isSubmissionEnabled} onClick={onClick}>
+            Next
+          </Button>
+        </div>
+      </div>
+      <div className="hidden absolute md:flex md:flex-row md:justify-end bottom-12 left-1/2 -translate-x-1/2 w-3/4 h-14">
+        <Button disabled={!isSubmissionEnabled} onClick={nextPageCallback}>
+          Next
+        </Button>
+      </div>
```

Comment:
There's a bit of clipping with this button on my macbook air (not 1920x1080):
<img width="336" alt="image" src="https://user-images.githubusercontent.com/23221268/227702702-c71a9769-6184-48f0-81d4-4d6dde95269e.png">

Perhaps instead of absolutely positioning this, you could just reuse the above button (line 55), and on `md` screens give it a width equal to a value _slightly_ bigger than the location input? Put this width on the div above it, then you should be good.